src/Tribe/Main.php

```diff
@@ -210,6 +210,8 @@ public function load_assets() {
 				array( 'tribe-dependency-style', 'dependency.css' ),
 				array( 'tribe-pue-notices', 'pue-notices.js', array( 'jquery' ) ),
 				array( 'tribe-datepicker', 'datepicker.css' ),
+				array( 'tribe-ui-styles', 'tribe-ui-styles.css' ),
+				array( 'tribe-utilities-styles', 'tribe-utilities-styles.css' ),
```

Comment:
Other CSS declarations have `-style`, `-css`, or nothing appended to them. I'd suggest one of those approaches rather than pluralizing with `-styles`.